Update manual.adoc (#4997)
rust-analyzer/rust-analyzer: docs/user/manual.adoc

1. Rename, symlink or copy the `rust-analyzer` binary to `rls` and place it somewhere Builder can find (in `PATH`, or under `~/.cargo/bin`).
2. Enable the Rust Builder plugin.

==== GNOME Builder (Nightly)

> https://nightly.gnome.org/repo/appstream/org.gnome.Builder.flatpakref[GNOME Builder (Nightly)] has now native support for `rust-analyzer` out of the box. If the `rust-analyzer` binary is not installed GNOME Builder will ask for installing the binary for you when opening up the first time a Rust source file.

Reviewer: ```suggestion
https://nightly.gnome.org/repo/appstream/org.gnome.Builder.flatpakref[GNOME Builder (Nightly)] has now native support for `rust-analyzer` out of the box. If the `rust-analyzer` binary is not installed, GNOME Builder will offer to install it when opening a Rust source file.
```
